tests/xharness/.editorconfig

```diff
@@ -0,0 +1,216 @@
+[*.{props,targets}]
+indent_style = tab
+indent_size = 4
+
+[*.{csproj}]
+indent_style = space
+indent_size = 2
+
+# C# files
+[*.cs]
+
+#### Core EditorConfig Options ####
+
+# Indentation and spacing
+indent_style = tab
+indent_size = 4
+tab_width = 4
+
+# New line preferences
+end_of_line = crlf
```

Comment:
This is wrong, we don't want `crlf` line endings in .cs files. This is either `lf` or nothing at all.